src/optimistic-oracle-v2/implementation/ManagedOptimisticOracleV2.sol

```diff
@@ -57,9 +58,9 @@ contract ManagedOptimisticOracleV2 is
     AccessControlDefaultAdminRulesUpgradeable,
     MultiCaller
 {
-    struct MaximumBond {
+    struct BondCapInfo {
         IERC20 currency;
-        uint256 amount;
+        uint256 capAmount;
```

Comment:
`capAmount` seem confusing when used for `minimumBonds`. I think its better to have the same struct hold both minimum and maximum amounts, also see the other comment on checking the consistency of these values.